Packs/TrendMicroDeepSecurity/README.md

```diff
@@ -1,4 +1,8 @@
-Trend Deep Security enables you to defend your system against network and application attacks. It can detect suspicious changes in your system, shield your servers from vulnerabilities, and enforce compliance. 
+# Trend Micro Deep Security
+
+This pack includes Cortex XSIAM content. 
```

Comment:
I assume this pack also has XSOAR content since you have an XSOAR section in this readme. So I think this sentence needs to be tagged for XSIAM only. WDYT?